recipes-core/ovmf/ovmf_git.bbappend

```diff
@@ -1,6 +1,16 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# xcp-ng patches taken from:
```

Comment:
It's a little weird the commit message says "Taken from XAPI", but the patches are prefixed xcp-ng.  Yes, I realized xapi is the upstream for xcp-ng.

Reply:
Joel and I are honestly a little confused ourselves. xcp-ng was the original source of the patches as XAPI hadn't upstreamed their solution yet when we started this work. Now that varstored is in XAPI proper, there should theoretically be a corresponding set of changes in ovmf, but we couldn't find any, neither in upstream OVMF nor in XAPI's repos. Their README points to a fork of edk2 that isn't accessible anymore. Not sure what to make of it.

I can rename these to `xapi-`, but the patches will still technically be from xcp-ng.

Reply:
I'd leave the file names and change the commit message.